datasets/dataforgood-fb-hrsl.yaml

```diff
@@ -1,10 +1,10 @@
 Name: High Resolution Population Density Maps + Demographic Estimates by CIESIN and Facebook
 Description: |
-  The population of the world, allocated to 1 arcsecond blocks. This refines
-  [CIESIN’s Gridded Population of the World](https://sedac.ciesin.columbia.edu/data/collection/gpw-v4)
+  Population data for 134 countries, allocated to 1 arcsecond blocks and provided in a combination of CSV
```

Comment:
Only edit I’d make here is to remove the country count and say something like “Population data for selected countries”.

Reply:
Great! Changed it to say "Population data for a selection of countries".